### ISSUE TITLE
Window position not working

### ISSUE THREAD

**Issue by micyoshua** [beeware/toga#1339]
**Describe the bug**
Setting the `position` of `toga.MainWindow` doesn't seem to have any effect, both in windows and linux.

**To Reproduce**
```
#sample.py
import toga

class Sample(toga.App):
    def startup(self):
        self.main_window = toga.MainWindow(title=self.name, position=(300, 200))

        self.main_window.content = toga.Box()
        self.main_window.show()

def main():
    return Sample('Sample', 'org.beeware.sample')


if __name__ == '__main__':
    main().main_loop()
```
`python -m sample`

**Expected behavior**
The window is displayed at the specified position.

**Actual Result**
In linux (gtk), it is positioned at (0, 0). In windows, it is positioned at (100, 100), but sometimes also randomly in different positions.

**Environment:**
 - Operating System: Debian 10 and Windows 10
 - Python version: 3.8+
 - Software versions:
   - Briefcase: `0.3.5`
   - Toga: `0.3.0.dev28`

**Comment by saroad2** [comment]
I think this was solved for windows in #1461

**Comment by freakboy3742** [comment]
Agreed; closing.
